climakitae/core/data_export.py

```diff
@@ -105,19 +105,22 @@ def _estimate_file_size(data: xr.DataArray | xr.Dataset, format: str) -> float:
     float
         estimated file size in gigabytes
     """
-    if format == "NetCDF" or format == "Zarr":
-        data_size = data.nbytes
-        buffer_size = 100 * 1024 * 1024  # 100 MB for miscellaneous metadata
-        est_file_size = data_size + buffer_size
-    elif format == "CSV":
-        # Rough estimate of the number of chars per CSV line
-        # Will overestimate uncompressed size by 10-20%
-        chars_per_line = 150
-
-        if isinstance(data, xr.core.dataarray.DataArray):
-            est_file_size = data.size * chars_per_line
-        elif isinstance(data, xr.core.dataset.Dataset):
-            est_file_size = prod(data.sizes.values()) * chars_per_line
+    match format:
+        case "NetCDF" | "Zarr":
+            data_size = data.nbytes
+            buffer_size = 100 * 1024 * 1024  # 100 MB for miscellaneous metadata
+            est_file_size = data_size + buffer_size
+        case "CSV":
+            # Rough estimate of the number of chars per CSV line
+            # Will overestimate uncompressed size by 10-20%
+            chars_per_line = 150
+
+            match data:
+                case data if isinstance(data, xr.core.dataarray.DataArray):
+                    est_file_size = data.size * chars_per_line
+                case data if isinstance(data, xr.core.dataset.Dataset):
+                    est_file_size = prod(data.sizes.values()) * chars_per_line
```

Comment:
proper formatting for type matching is:
```python
match data:
    case xr.DataArray():
        est_file_size = data.size * chars_per_line
    case xr.Dataset():
        est_file_size = prod(data.sizes.values()) * chars_per_line
```
and generally the most common option should be listed first. No idea what that should be in this case, but just a heads up.

Reply:
Fixed in [7a4f30b](https://github.com/cal-adapt/climakitae/pull/564/commits/7a4f30b74fd8396c8d5ed0854623b4c3c03e5bac)